src/api/listener.ts

```diff
@@ -0,0 +1,74 @@
+import { storeToRefs } from "pinia";
+import { listen } from "@tauri-apps/api/event";
+import { ref } from "vue";
+import { ManagedSerialPortsEvent } from "@/events/managed-serial-ports";
+import { PacketEvent } from "@/events/packet";
+import { PacketData } from "@/models/intern/packet-data";
+import { useAppStore } from "@/stores/app";
+import { useTheme } from "vuetify";
+
+export type RemoveListenerFunction = () => void;
+
+export const useListener = (app = useAppStore()) => {
+  const { managedSerialPorts } = storeToRefs(app);
+  const { addPacket, getSerialPorts } = app;
+  const theme = useTheme();
+
+  const themeChangedEventListener = ref<RemoveListenerFunction>();
+  const serialPortEventListener = ref<RemoveListenerFunction>();
+  const serialPortPacketEventListener = ref<RemoveListenerFunction>();
+
+  const setupListeners = async () => {
+    themeChangedEventListener.value = await listen(
+      "tauri://theme-changed",
```

Comment:
In `@tauri-apps/api/event` sind alle Tauri events als Constanten. Wir können `"tauri://theme-changed"` durch die Konstante ersetzen.

Reply:
welche Konstante?

Reply:
jo wird gemacht ...